thirdparty/s3.go

```diff
@@ -10,10 +10,13 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 )
 
-//PresignExpireTime sets the amount of time the link is live before expiring
-const PresignExpireTime = 24 * time.Hour
+const (
+	//PresignExpireTime sets the amount of time the link is live before expiring
```

Comment:
nit: space between `//` and start of comment. Also, period at the end of a full sentence.